src/components/Homepage/HomepageStatistics.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+	<section class="statistics section">
+		<div class="row">
+			<h1
+				v-html="statsHeadline"
+				class="small-12 large-6 columns statistics__header huge green-emphasis"
+			>
+			</h1>
+			<span class="statistics__video-wrapper small-12 large-6 columns">
+				<video
+					:src="statsVideo.file.url"
+					class="statistics__video-wrapper--video"
+					autoplay
+					loop
+					muted
+					playsinline
+				></video>
+			</span>
+		</div>
+		<div class="row statistics__stat-block">
+			<div
+				v-for="(statBlock, index) in statsBlockText"
+				:key="statBlock.key"
+				class="small-12 medium-4 columns"
+			>
+				<kv-contentful-img
+					v-if="statBlock.image.url"
+					:class="'statistics__stat-block--icon statistics__stat-block--icon-' + index"
+					:contentful-src="statBlock.image.url"
+					:alt="statBlock.image.description"
+					fallback-format="png"
+					:width="50"
+				/>
+				<p
+					class="statistics__stat-block--stat green-emphasis"
+					v-html="formattedTextStrings[index]"
+				>
+				</p>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import numeral from 'numeral';
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import whyKivaQuery from '@/graphql/query/whyKivaData.graphql';
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+export default {
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			repaymentRate: '',
+			totalLoansInDollars: '',
+			numCountries: '',
+			numLenders: '',
+		};
+	},
+	apollo: {
+		query: whyKivaQuery,
+		preFetch: false,
+		result({ data }) {
+			this.repaymentRate = _get(data, 'general.kivaStats.repaymentRate');
+			this.totalLoansInDollars = _get(data, 'general.kivaStats.amountFunded');
+			this.numCountries = _get(data, 'general.kivaStats.numCountries');
+			this.numLenders = _get(data, 'general.kivaStats.numLenders');
+		}
+	},
+	components: {
+		KvContentfulImg,
+	},
+	props: {
+		content: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		statsText() {
+			return this.content?.contents?.find(({ key }) => key.indexOf('homepage-statistics-headline-text') > -1);
+		},
+		statsHeadline() {
+			return this.statsText.headline;
+		},
+		statsBlockText() {
+			// eslint-disable-next-line max-len
+			const allStatsText = this.content?.contents?.filter(({ key }) => key.indexOf('homepage-statistics-text') > -1);
+			return allStatsText?.map((block, index) => ({
+				key: block.key || index,
+				copy: documentToHtmlString(block.bodyCopy ?? ''),
+				image: this.statsIcons[index],
+			}));
+		},
+		statsIcons() {
+			const icons = this.content?.media ?? [];
+			return icons.map(image => ({
+				description: image?.description ?? '',
+				title: image?.title ?? '',
+				url: image?.file?.url ?? ''
+			}));
+		},
+		statsVideo() {
+			// Grabbing the 2x video from contentful.
+			// this.content?.media[3] is the 1x video
+			const video = this.content?.media[4] ?? [];
```

Comment:
It would be safer to grab the video by using its contentful title instead of its position in the array. Right now if the  
 order where changed in Contentful this could break.

Reply:
Also added more specificity to the statsIcons. Thanks.